db_stress_tool/db_stress_test_base.cc

```diff
@@ -2165,5 +2172,67 @@ void StressTest::Reopen(ThreadState* thread) {
           num_times_reopened_);
   Open();
 }
+
+void StressTest::MaybeStartTrace() {
+#ifndef ROCKSDB_LITE
+  if (FLAGS_trace_path.empty()) {
+    return;
+  }
+  trace_enabled_ = true;
+
+  if (FLAGS_use_txn) {
+    fprintf(stdout, "--trace_path with --use_txn is not supported.\n");
+    trace_enabled_ = false;
+  }
+
+  if (FLAGS_use_blob_db) {
+    fprintf(stdout, "--trace_path with --use_blob_db is not supported.\n");
+    trace_enabled_ = false;
+  }
+
+  if (trace_env == nullptr) {
+    fprintf(stdout, "trace_env is null.  Disable --trace.\n");
+    trace_enabled_ = false;
+  }
+
+  if (trace_enabled_ == false) {
+    fprintf(stdout, "--trace_path is disabled.\n");
+    return;
+  }
+
+  EnvOptions trace_env_options(options_);
+  std::unique_ptr<TraceWriter> trace_writer;
+
```

Comment:
Should we set a trace file limit here?

Reply:
Sure, I can add an option here.

Reply:
It seems only the BlockCacheTrace has the file limit while TraceWriter does not.  Am I right at this part?

Reply:
WIthout setting it it will use the default 64GB limit.